### PR TITLE
[node-manager] Alert about yandex ru-central-c zone deprecation

### PR DESCRIPTION
**Pull request by mvasl**
## Description

If cluster has nodes on Yandex.Cloud availability zone `ru-central1-c`, alert will be fired asking user to migrate to zone `ru-centra1-d`

## Why do we need it, and what problem does it solve?

Yandex.Cloud [deprecated](https://cloud.yandex.com/en-ru/docs/overview/concepts/ru-central1-c-deprecation) `ru-central1-c` availability zone, users are encouraged to migrate to `ru-central1-d`

## Checklist
- [x] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: node-manager
type: feature
summary: Alert about yandex ru-central-c zone deprecation
impact_level: default
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @mvasl. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]
move all to 030-cloud-provider-yandex please

File: modules/030-cloud-provider-yandex/monitoring/prometheus-rules/deprecated_availability_zone.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+        severity_level: "9"
+      annotations:
+        plk_markup_format: markdown
+        plk_protocol_version: "1"
+        plk_create_group_if_not_exists__cluster_has_node_groups_deprecation_alerts: "AvailabilityZonesDeprecationAlerts,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
+        plk_grouped_by__cluster_has_node_groups_deprecation_alerts: "AvailabilityZonesDeprecationAlerts,tier=cluster,prometheus=deckhouse,kubernetes=~kubernetes"
+        summary: NodeGroup {{ $labels.node_group }} contains Nodes with deprecated availability zone.
+        description: |
+          Availability zone `ru-central1-c` is deprecated by Yandex.Cloud.
+          You should migrate your Nodes to the new `ru‑central1‑d` zone.
```
Comment (name212) [inline modules/030-cloud-provider-yandex/monitoring/prometheus-rules/deprecated_availability_zone.yaml:18]: We don't support `d` zone yet. Please change to `ru‑central1‑a` or `ru‑central1‑b`

**Review by name212 — COMMENTED** [review]
comment

File: modules/030-cloud-provider-yandex/monitoring/prometheus-rules/deprecated_availability_zone.yaml (hunk 1 above)
Comment (mvasl) [reply modules/030-cloud-provider-yandex/monitoring/prometheus-rules/deprecated_availability_zone.yaml]: Done

**Review by name212 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**